packages/tradershub/src/features/cfd/constants.tsx

```diff
@@ -83,7 +77,7 @@ export const MarketTypeDetails: TMarketTypeDetails = {
     },
 };
 
-export const PlatformDetails: TPlatformDetails = {
+export const PlatformDetails = {
```

Comment:
Removing this type cause typescript is smart enough to infer the correct types

![Screenshot 2023-12-26 at 12 47 13 PM](https://github.com/binary-com/deriv-app/assets/104053934/ce857f16-c76f-4dfb-9d8c-2ba34ad34b25)